core/deepmind.py

```
@@ -407,7 +412,22 @@ async def organize_and_inject_memories(
+        try:
+            memory_sql_manager = self.plugin_context.get_component("memory_sql_manager")
+            if memory_sql_manager is not None:
+                self.user_profile_service.set_memory_sql_manager(memory_sql_manager)
+            memory_scope = await self.plugin_context.resolve_memory_scope_from_event(event)
+            await self.user_profile_service.refresh_session_profiles(
+                session_id=session_id,
+                chat_records=unprocessed_chat_records,
+                fallback_sender_id=str(getattr(event, "sender_id", "") or ""),
+                memory_scope=memory_scope,
+            )
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)